std/string.d

```diff
@@ -2033,6 +2040,216 @@ S[] splitLines(S)(S s, in KeepTerminator keepTerm = KeepTerminator.no) @safe pur
     });
 }
 
+/***********************************
+ *  Split an array or slicable range of characters into a range of lines
+    using $(D '\r'), $(D '\n'),
+    $(D "\r\n"), $(XREF uni, lineSep), and $(XREF uni, paraSep) as delimiters.
+    If $(D keepTerm) is set to $(D KeepTerminator.yes), then the delimiter
+    is included in the slices returned.
+
+    Does not throw on invalid UTF; such is simply passed unchanged
+    to the output.
+
+    Does not allocate memory.
```

Comment:
That's actually a common usage when describing how something works. The subject is implied.